en/README.md

```diff
@@ -0,0 +1,1138 @@
+# Arm Plugin Interface 2.0 Specification(2015/04/16)
+
+## Overview
+
+Hatohol Arm Plugin Interface (HAPI) 2.0 is protocol for information exchange between Hatohol server and Monitoring server plugin.
```

Comment:
I think that `the` is needed like this:

Hatohol Arm Plugin Interface (HAPI) 2.0 is **the** protocol ...